broadcast/src/linked/mod.rs

```diff
@@ -0,0 +1,663 @@
+//! Secure, ordered broadcast protocol with linked messages for consistent network
+//! dissemination using cryptographic proofs, sequential linking, and robust state
+//! management.
+//!
+//! # Core Concepts
+//!
+//! - **Sequential Linking**: Every broadcast message contains a cryptographic reference
+//!   to the previous message, forming an immutable chain.
+//!
+//! - **Threshold Signatures**: Validators produce partial signatures that combine into a
+//!   threshold signature, ensuring a quorum verifies each message.
+//!
+//! - **Epochs**: The protocol is divided into epochs. Each epoch defines the active set of
```

Comment:
I don't think we should be mentioning consensus here (there is no consensus in this crate). I think we can just say it handles reconfiguration (while chunks are inflight via rebroadcast).